db/orders.js

```diff
@@ -67,33 +92,63 @@ const getAllOrders = async () => {
   }
 }
 
-const getOrdersByUser = async ({id}) => {
+const getOrdersByUser = async (id) => {
   try {
-    const {rows: [user]} = await client.query(`
-    SELECT * FROM users
-    WHERE id = $1;
+    const {rows: [userOrder]} = await client.query(`
+    SELECT users.username, users.email, orders.status, orders."datePlaced", op.price, op.quantity, p.description, p.price 
+    FROM users
+    JOIN orders ON users.id = orders."userId"
```

Comment:
You likely will need to do some data massaging to put products nested in the (each) order